### ISSUE TITLE
Integrated with a video player?

### ISSUE THREAD

**Issue by dgoryeo** [davabase/transcriber_app#16]
Hi, Just found this app. Thanks for the good work. Is the transcriber integrated with a video player --similar to shown in the main image in readme? Thanks.

**Comment by davabase** [comment]
No, the Transcriber is a standalone process. It has a setting to make the window transparent so it can easily be placed on top of other windows like in the README.
